### PR TITLE
 stub everything

### PR DESCRIPTION
**Pull request by deads2k**
This produces a command and some rough resources that probably won't work, but gets the repo started.

/assign @hexfusion

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+  - winterm
+- name: github.com/beorn7/perks
+  version: 3ac7bf7a47d159a033b107610db8a1b6575507a4
+  subpackages:
+  - quantile
+- name: github.com/certifi/gocertifi
+  version: ee1a9a0726d2ae45f54118cac878c990d4016ded
+- name: github.com/coreos/etcd
+  version: 420a452267a7ce45b3fcbed04d54030d69964fc1
+  subpackages:
```
Comment (hexfusion) [inline glide.lock]: so this is etcd 3.2.24 openshift 4.0 is using 3.3.x. (3.3.10) Now k8s 1.11 brought in the v3.3.x etcd client I am just wondering if we should bump this as well?

https://github.com/kubernetes/kubernetes/issues/61326#issuecomment-374737257

Comment (hexfusion) [reply glide.lock]: well no that didn't make it until 1.13 https://github.com/kubernetes/kubernetes/pull/69322 nmd.

**Comment by deads2k** [comment]
I'm going to merge this so that we can start the CI process.  Feel free to continue making comments and we can take time tomorrow to explain what it does

File: bindata/v3.11.0/etcd/svc.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    service.alpha.openshift.io/serving-cert-secret-name: serving-cert
+    prometheus.io/scrape: "true"
+    prometheus.io/scheme: https
+spec:
+  selector:
+    etcd: "true"
+  ports:
+  - name: https
+    port: 443
+    targetPort: 10257
```
[the remark this answers is not in the document]
Comment (sttts) [reply bindata/v3.11.0/etcd/svc.yaml]: for etcd I would not map the port, but just use the normal 2379 and 2380 one-to-one.

File: manifests/image-references [hunk 3]
```diff
@@ -0,0 +1,12 @@
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+spec:
+  tags:
+  - name: cluster-etcd-operator
+    from:
+      kind: DockerImage
+      name: docker.io/openshift/origin-cluster-etcd-operator:v4.0
+  - name: hyperkube
```
[the remark this answers is not in the document]
Comment (sttts) [reply manifests/image-references]: @tnozicka can get you into the image building pipeline. He is probably already on it.

[the remark this answers is not in the document]
Comment (hexfusion) [reply manifests/image-references]: FTR we are working on this. https://github.com/openshift/release/pull/2758

**Pull request merged**